etc/base.yaml

```diff
@@ -270,11 +270,10 @@ radar_rest_sources_backend:
     garmin:
       enable: "false"
 
-# --------------------------------------------------------- 20-grafana.yaml ---------------------------------------------------------
+# --------------------------------------------------------- 20-dashboard.yaml ---------------------------------------------------------
 
 timescaledb_username: postgres
-timescaledb_db_name: grafana-metrics
-grafana_metrics_username: postgres
+timescaledb_db_name: data-dashboard
```

Comment:
Changing the database name won't be backwards compatible, I suggest not changing it. If it's necessary we need to have a migrated guide during release.

Reply:
I decided to have data-dashboard-backend run in a dedicated database. Now, we are backwards compatible again.